sale_stock_ux/__manifest__.py

```diff
@@ -30,19 +30,16 @@
     ],
     'depends': [
         'sale_stock',
-        'stock_usability',
-        'sale_usability',
+        'sale_ux',
```

Comment:
> +        'sale_ux',

did you mean stock_ux?

Reply:
no, I mean "exits ux", "ux stock" is not necessary now because I deleted the part of the code that used functionality of that module